docker/README.md

```diff
@@ -0,0 +1,15 @@
+This is the configuration and usage instructions for the docker daemon on BeagleBone.
+
+ * [Instalation instructions](https://docs.docker.com/engine/installation/linux/docker-ce/debian) for docker on arm.
+ 
+ * DNS setup with JSI DNS servers for docker daemon: Place the following line to `/etc/docker/daemon.json`:
+
+    `{"dns": ["193.2.4.247", "193.2.4.248"]}`
+
+ * Set permissions for docker group to use serial ports in file `/etc/udev/rules.d/60-omap-tty.rules`:
```

Comment:
There are already udev rules for the serial port on the stock system. Devices are owned by the `dialout` group. Is it possible to use that with instead? (e.g. by add the `docker` user to the `dialout` group).